docs/local-dev/debugging.md

```diff
@@ -71,3 +71,19 @@ In your `launch.json`, add a new entry with the following,
     "console": "integratedTerminal"
 },
 ```
+
+### WebStorm
+
+This section describes the process for enabling run configurations for Backstage in WebStorm.
+Run configurations enable the use of debugging functionality such as steppers and breakpoints.
+
+1. Select `Edit Configurations` in the `Run` dropdown menu. Click the plus sign to add a new configuration, then select `Node.js`.
+2. In `Working directory`, input `{PROJECT_DIR}/packages/backend`.
```

Comment:
What is `{PROJECT_DIR}`? Are you trying to say thats the path to their project? If so then lets put a note at the end:

> Note: Replace `{PROJECT_DIR}` with the path to your projct